defaults/main.yml

```diff
@@ -5,3 +5,7 @@ theo_agent_user: theo-agent
 theo_agent_config_dir: /etc/theo-agent
 theo_agent_config_path: "{{ theo_agent_config_dir }}/config.yml"
 theo_agent_cache_dir: /var/cache/theo-agent
+theo_verify_signature: false
+theo_public_key: ""
+theo_public_key_path: "{{ theo_agent_config_dir }}/public.pem"
+theo_config_public_key_path: ""
```

Comment:
Should these be `theo_` o `theo_agent_` ?

Reply:
yes, you're right